src/scss/components/secnav/mixins/_secNavControl.scss

```diff
@@ -8,16 +8,16 @@
     box-shadow: $focus-glow;
 
     &::-ms-check {
-      border-color: $cyan-700;
+      border-color: var(--hxSecNav-focus-ie-borderColor, $cyan-700);
```

Comment:
Please update to Color Palette v2.0 color specs.

Reply:
Changes have been done, Please review